src/test/kotlin/com/exactpro/th2/common/event/TestEvent.kt

```diff
@@ -305,6 +309,42 @@ class TestEvent {
         )
     }
 
+    @Test
+    fun `book mismatch between attached message and event`() {
```

Comment:
Test for event with different book and scope in event and parent can't be written because Event.setParentId is private.
User define book and scope via to... methods